Snowflake.Data.Tests/App.config

```diff
@@ -19,11 +19,17 @@ Copyright (c) 2012-2017 Snowflake Computing Inc. All rights reserved.
         <!-- <header value="[DateTime]  [Thread]  [Level]  [ClassName] Message&#13;&#10;" /> -->
         <conversionPattern value="[%date] [%t] [%-5level] [%logger] %message%newline" />
       </layout>
+      <appender name="ConsoleAppender" type="log4net.Appender.ConsoleAppender">
+        <layout type="log4net.Layout.PatternLayout">
+          <conversionPattern value="%date [%thread] [%-5level] [%method:%line] - %message%newline" />
+        </layout>
+      </appender>
     </appender>
 
     <root>
       <level value="ALL" />

```

Comment:
If we configure console logging for tests, maybe we can change here to info or, even better, to warn? If anyone needs more logging, always can be changed.